web_tool_script_2.R

```diff
@@ -209,14 +209,6 @@ if (file.exists(bonds_inputs_file)) {
   }
 }
 
-# provide parameters for stress test
-invisible(set_portfolio_parameters(file_path = fs::path(par_file_path, paste0(portfolio_name_ref_all, "_PortfolioParameters.yml"))))
-# set environment variable for stress test data path
-options("ST_DATA_PATH" = stress_test_data_location)
-# run 2dii stress test
-source(file.path(stress_test_path, "web_tool_stress_test.R"))
-# run stress test with external scenarios (IPR)
-source(file.path(stress_test_path, "web_tool_external_stress_test.R"))
 
 rm(port_raw_all_eq)
 rm(port_raw_all_cb)
```

Comment:
I see the stress test code moves from before to after some global variables are removed `port_raw_all_eq`, `port_raw_all_cb`, etc.) Is this okay @jacobvjk? In other words, does the stress test code work without those variables?

Reply:
web tool scripts 1-3 all run independently in separate environments, i.e. webtool script 3 cannot rely on anything being set in webtool script 1 or 2... so all necessary environment variables should be set at the top of each script

Reply:
The stress test reads the pacta results from `working_dir/40_Results`. Whether or not any intermediate pacta results are removed from the environment in the meantime should pose no problem. As long as all paramters and paths are set, it should be good